packages/@aws-cdk/aws-ec2-alpha/lib/route.ts

```diff
@@ -402,6 +436,106 @@ export class NatGateway extends Resource implements IRouteTarget {
   }
 }
 
+/**
+ * Creates a peering connection between two VPCs
+ * @resource AWS::EC2::VPCPeeringConnection
+ */
+export class VPCPeeringConnection extends Resource implements IRouteTarget {
+
+  /**
+   * The type of router used in the route.
+   */
+  readonly routerType: RouterType;
+
+  /**
+   * The ID of the route target.
+   */
+  readonly routerTargetId: string;
+
+  /**
+   * The VPC peering connection CFN resource.
+   */
+  public readonly resource: CfnVPCPeeringConnection;
+
+  constructor(scope: Construct, id: string, props: VPCPeeringConnectionProps) {
+    super(scope, id);
+
+    this.routerType = RouterType.VPC_PEERING_CONNECTION;
+
+    const isCrossAccount = Stack.of(props.requestorVpc).account !== Stack.of(props.acceptorVpc).account;
```

Comment:
i think this check should now be modified to new value added to the interface i.e. `props.acceptorVpc.ownerAccountId`

Reply:
Good catch! [9f96a92](https://github.com/aws/aws-cdk/pull/31645/commits/9f96a92b19a2a4de86c641bdd64ee0b84c501082)